src/clockodo.ts

```diff
@@ -447,7 +447,7 @@ export class Clockodo {
   async startClock(
     params: Params<
       Pick<TimeEntry, typeof REQUIRED.START_CLOCK[number]> & {
-        billable: ClockingTimeEntryBillability;
+        billable: ClockingTimeEntryBillability | undefined;
```

Comment:
I think I'd make that property optional (and would add a short explanation to it):

```suggestion
        /**
         * Billability of the time entry that is about to be created.
         * Omit it if you want to use the project's default.
         */
        billable?: ClockingTimeEntryBillability;
```